### ISSUE TITLE
qml module x not found

### ISSUE THREAD

**Issue by Lukalukito** [contour-terminal/contour#1589]
### Contour Terminal version

0.4.4

### Installer source

something else (please specify below)

### Operating System

arch linux

### Architecture

x86-64

### Steps to reproduce

launch contour with QT_STYLE_OVERRIDE variable set. in my case i have it set to kvantum

### Expected Behavior

contour opens

### Actual Behavior

getting a warning:
```
Warning: QQmlApplicationEngine failed to load component ((null):0, (null))
Warning: qrc:/contour/ui/main.qml: module "kvantum" is not installed (qrc:/contour/ui/main.qml:4294967295, (null))
```
and contour doesn't open

### Additional notes

when i unset QT_STYLE_OVERRIDE var contour opens without warnings, but I want to have that var set to kvantum so that in other qt programs, like file manager(pcmanfm-qt for me) transparency and blurring works. if i set that var to something random, it just says invalid style override and ignores it, also lists valid style overrides, but i tried all of them and it still show that warning.

**Comment by Yaraslaut** [comment]
Hi @Lukalukito, thanks for the bug report
It seems that this issue is not connected to kvantum per se, something is wrong with handling of style on qml side, i get similar error if i use "existing" style on my system like Windows.

So if i try to run with some non existing style i get message 
```
Warning: QApplication: invalid style override 'asdf' passed, ignoring it.
        Available styles: Windows, Fusion ((null):0, (null))

```
but `contour` starts only if `QT_STYLE_OVERRIDE` equal to `Fusion`, if i try to run with Windows i get
```
Warning: QQmlApplicationEngine failed to load component ((null):0, (null))
Warning: file:///home/yaraslau/repo/contour/src/contour/ui/main.qml: module "Windows" is not installed (file:///home/yaraslau/repo/contour/src/contour/ui/main.qml:4294967295, (null))
```

**Comment by Yaraslaut** [comment]
Hm, from https://doc.qt.io/qt-6/qtquickcontrols-styles.html#windows-style it looks like Windows style is not something that i am supposed to use on linux 
```
Note: this style is only available for applications running on Windows.
```

**Comment by Lukalukito** [comment]
Yeah, really silly workaround I found was to run `set -e QT_STYLE_OVERRIDE; contour; set QT_STYLE_OVERRIDE [whatever it was before]` or something like that on fish shell (set is similar to export on other shells and set -e removes a variable, i think permanently and that's why i need to set it again after contour exits). This command works which is really wierd
